buf-gogo.gen.yaml

```diff
@@ -2,4 +2,6 @@ version: v1
 plugins:
   - name: gogofast
     out: ./gogogen
-    opt: plugins=grpc
+    opt:
+      - plugins=grpc
+      - Mgoogle/protobuf/wrappers.proto=github.com/gogo/protobuf/types
```

Comment:
This makes sure we use the gogo-backed types for wrappers.proto. We "lucked out" in the current usage, as events.proto sets the options to turn the fields into pointers instead.